src/Methods/Message.php

```diff
@@ -18,13 +18,16 @@ trait Message
      *
      * <code>
      * $params = [
-     *       'chat_id'                   => '',  // int|string - Required. Unique identifier for the target chat or username of the target channel (in the format "@channelusername")
-     *       'text'                      => '',  // string     - Required. Text of the message to be sent
-     *       'parse_mode'                => '',  // string     - (Optional). Send Markdown or HTML, if you want Telegram apps to show bold, italic, fixed-width text or inline URLs in your bot's message.
-     *       'disable_web_page_preview'  => '',  // bool       - (Optional). Disables link previews for links in this message
-     *       'disable_notification'      => '',  // bool       - (Optional). Sends the message silently. iOS users will not receive a notification, Android users will receive a notification with no sound.
-     *       'reply_to_message_id'       => '',  // int        - (Optional). If the message is a reply, ID of the original message
-     *       'reply_markup'              => '',  // string     - (Optional). Additional interface options. A JSON-serialized object for an inline keyboard, custom reply keyboard, instructions to remove reply keyboard or to force a reply from the user.
+     *       'chat_id'                   => '',  // int|string           - Required. Unique identifier for the target chat or username of the target channel (in the format "@channelusername")
+     *       'text'                      => '',  // string               - Required. Text of the message to be sent
+     *       'parse_mode'                => '',  // string               - (Optional). Send Markdown or HTML, if you want Telegram apps to show bold, italic, fixed-width text or inline URLs in your bot's message.
+     *       'disable_web_page_preview'  => '',  // bool                 - (Optional). Disables link previews for links in this message
+     *       'disable_notification'      => '',  // bool                 - (Optional). Sends the message silently. iOS users will not receive a notification, Android users will receive a notification with no sound.
+     *       'reply_to_message_id'       => '',  // int                  - (Optional). If the message is a reply, ID of the original message
+     *       'reply_markup'              => '',  // InlineKeyboardMarkup - (Optional). Additional interface options. A JSON-serialized object for an inline keyboard, custom reply keyboard, instructions to remove reply keyboard or to force a reply from the user.
+     *                                           // ReplyKeyboardMarkup
```

Comment:
Is this worth while?

Adding the 4 different type of reply_markup for easy ref? Would have to do it for a lot of methods. Just did it here to show you an example.

Reply:
Inline them please. Too verbose. You can rewrite the docs comment to a brief one since the new objects do the job, and they need not pass the JSON-serialized object.

Reply:
New example now in `sendMessage`. If you think it's ok let me know and I might do a search and replace for the other methods too..